bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/general/GeneralRule.java

```diff
@@ -103,61 +102,49 @@ public IP2Problems check(ObservationElements elements)  {
 
         private static final String WFS_EMPTY_NAME_TEMPLATE = "%s name field should not be empty";
         private static final String P2_PREFERRED = "PWFS2 is the preferred peripheral wavefront sensor.";
-        private static final String NO_AO_OTHER = "Altair cannot use both %s and %s.";
+        private static final String NO_AO_OTHER = "Altair cannot use %s and %s.";
```

Comment:
This was changed because this list may now contain more than two guiders due to Altair now supporting multiple guiders. The first %s may become [AOWFS, GMOS OI] and the second something else, although I don't think this P2 warning can occur in new programs.